chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/CertifiedProductDetailsDTO.java

```diff
@@ -240,6 +250,46 @@ public void setDecertificationDate(Date decertificationDate) {
         this.decertificationDate = Util.getNewDate(decertificationDate);
     }
 
+    public String getRwtPlanUrl() {
+        return rwtPlanUrl;
+    }
+
+    public void setRwtPlanUrl(String rwtPlanUrl) {
+        this.rwtPlanUrl = rwtPlanUrl;
+    }
+
+    public Date getRwtPlanSubmissionDate() {
+        return rwtPlanSubmissionDate;
+    }
+
+    public void setRwtPlanSubmissionDate(Date rwtPlanSubmissionDate) {
+        this.rwtPlanSubmissionDate = rwtPlanSubmissionDate;
```

Comment:
Same questions in this class about the Date thing.

Reply:
Same as above